### PR TITLE
[FIXED JENKINS-36299] add 'all passing' view to test display

### PR DESCRIPTION
**Pull request by kzantow**
Related to issue # JENKINS-36299. 

Summary of this pull request: 
. Add encouraging 'all passing' view for test results

image

Also needs: https://github.com/jenkinsci/jenkins-design-language/pull/77

@reviewbybees

### CODE REVIEW

**Comment by cliffmeyers** [comment]
This LGTM to me. Is there any chance of adding a unit test for TestResults.jsx that asserts the correct state is rendered?

**Comment by cliffmeyers** [comment]
Adding a 🐝  anyways, will let @kzantow make the call as to whether the unit test is too much a hassle.

File: blueocean-dashboard/src/test/js/testResult-spec.js [hunk 1]
```diff
@@ -6,12 +6,9 @@ import { shallow } from 'enzyme';
 import TestResults from '../../main/js/components/testing/TestResults.jsx';
 
 describe("TestResults", () => {
-  let wrapper;
-  const
-    renderer = createRenderer();
+  const renderer = createRenderer();
```
Comment (cliffmeyers) [inline blueocean-dashboard/src/test/js/testResult-spec.js]: I think you can remove this from the test if you like.

**Comment by cliffmeyers** [comment]
Great to see more tests! :bee

**Comment by sophistifunk** [comment]
Is this component only shown when there are test results? What I mean is, it's never used to render a (no-fail + no-pass) situation?

**Comment by kzantow** [comment]
@sophistifunk yes, only used for the 'all passing', there's another thing that shows up if there are 0 tests.

**Comment by kzantow** [comment]
@michaelneale I'd like to get this merged, could do a quick review of this one?

**Comment by michaelneale** [comment]
:bee: I did test this wiht the kzantow/failure-project (I created an all passing branch).

**Pull request merged**